app/views/content/landing/how-much-do-teachers-get-paid.md

```diff
@@ -10,7 +10,6 @@ content:
 image: "static/images/content/hero-images/science.jpg"
 colour: "pink-blue"
 layout: "layouts/minimal"
-talk_to_us: false
```

Comment:
Related to the comments on the `/landing/campus-mailing-list` page, does there need to be a little more padding above the footer generally @jenhadfield-dfe ?

![image](https://github.com/user-attachments/assets/c80f4016-6952-4440-9dcb-079c198a4e31)